<!--#######################################-->
<!-- Copyright (c) 2023  Dong Zhaorui.     -->
<!-- All rights reserved.                  -->
<!-- Author: Dong Zhaorui 847235539@qq.com -->
<!-- Date  : 2023-11-21                    -->
<!--#######################################-->

<launch> 
    <node name="hex_livox_preprocess" pkg="hex_livox_preprocess" exec="hex_livox_preprocess" output="screen" >
        <!-- parameter -->
        <param from="$(find-pkg-share hex_livox_preprocess)/config/ros2/hex_livox_preprocess.yaml"/>
        <param name="max_count" value="20"/>
        <!-- subscribe -->
        <remap from="/in_string" to="/in" />
        <!-- publish -->
        <remap from="/out_string" to="/out" />
    </node>
</launch>
